osquery/tables/cloud/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@
 # the LICENSE file found in the root directory of this source tree.
 
 function(osqueryTablesCloudMain)
-  generateOsqueryTablesCloud()
+  if(NOT DEFINED PLATFORM_WINDOWS)
```

Comment:
I should change this to be `if(DEFINED PLATFORM_LINUX)` as these tables are not used on macOS either.